package-lock.json

```diff
@@ -1,24 +1,46 @@
 {
   "name": "ttfiles",
-  "version": "2.6.0",
-  "lockfileVersion": 1,
+  "version": "2.6.1",
+  "lockfileVersion": 2,
```

Comment:
re: your PR note about this - seems fine to me to be on version 2. We compile this locally and I'd imagine most of us are on newer node/npm versions as well anyway.